docs/product-installation-guide/src/main/asciidoc/clustering-artemis-activate-proc.adoc

```diff
@@ -1,4 +1,4 @@
-[id='clustering-artemis-activate-proc']
+[id='clustering-artemis-activate-proc_{context}']
 = Activating ActiveMQ JMS broker
 After you install Elasticsearch, you must activate ActiveMQ JMS broker on the main node of the cluster.
```

Comment:
so does it use the embedded JMS broker in EAP or there is a need to use standalone broker? if it's embedded isn't there some additional steps needed to make it clustered? just asking not sure....

Reply:
I've added a draft of the standalone section. If possible and practical, we only want to document the most relevant option.